web/src/api/config.ts

```diff
@@ -20,15 +20,18 @@
  * find current contact information at www.suse.com.
  */
 
-import * as l10n from "~/api/l10n/config";
-import * as storage from "~/api/storage/config";
-import * as product from "~/api/product/config";
+import * as l10n from "~/api/config/l10n";
+import * as storage from "~/api/config/storage";
 
 type Config = {
+  product?: Product;
   l10n?: l10n.Config;
   storage?: storage.Config;
-  product?: product.Config;
```

Comment:
this looks strange. I think @imobachgs should probably check it as I kind of expect we should have soon proper type for product and software config here.